scripts/release/packager/project.yml

```diff
@@ -0,0 +1,23 @@
+name: MapboxMapsPackager
+targets:
+  MapboxMaps:
```

Comment:
we might need to set a swift active compilation condition on this target to account for some conditional compilation we needed to add to eliminate warnings when building under SPM.

Reply:
Resolved since we're using `Mapbox.xcconfig`